NanoVNASaver/NanoVNASaver.py

```diff
@@ -36,6 +37,10 @@
 from .Touchstone import Touchstone
 from .Analysis import Analysis, LowPassAnalysis, HighPassAnalysis, BandPassAnalysis
 from .about import version as ver
+from PyQt5.QtWidgets import QApplication, QPushButton, QDesktopWidget
```

Comment:
I generally refer to the widgets as QtWidgets.QPushButton, for example - so these should probably be removed.

Reply:
done